style.css

```diff
@@ -326,6 +316,184 @@ nav {
   background-color: #fff;
 }
 
+/* ======================== Popup ======================== */
+.popup-works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 88px;
+  height: 0;
+}
```

Comment:
- [ ] In my opinion, it would be better to make sure your overlay background is blurred as shown on  the Figma design below :recycle: 

- Your design:
![Screenshot from 2023-02-10 12-31-01](https://user-images.githubusercontent.com/97834160/218070680-fb6aa80f-643d-4213-96f0-8612a4fba096.png)
- Figma:
![Screenshot from 2023-02-10 12-32-02](https://user-images.githubusercontent.com/97834160/218071373-6deb678d-ad66-44ee-af9f-a5256ff4eb81.png)


- [OPTIONAL]  I also suggest making the whole overlay scrollable instead of the popup itself :recycle:

Reply:
I have tried using the `backdrop-filter: blur(20px);` property for that but it doesn't work. Could you suggest some other ways to do that?

Reply:
- In my project, I usually create a container that will fill the whole screen and then give it a transparent and blur background,